packages/odyssey-prettier/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@okta/odyssey-prettier",
+  "version": "0.0.0",
+  "main": "src/index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "prettier": "2.4.0"
```

Comment:
do we need to pin here? Can we use a more flexible version like `^2.4.0` ?

Reply:
We dont, will update.